### PR TITLE
update podspec to add the Handler subdirectory

### PR DESCRIPTION
**Pull request by slizeray**
With the current pod spec the framework can't compile....The handler sub directory is missing

### CODE REVIEW

**Comment by slizeray** [comment]
@phimage Hello, is there any issue to merge this pull request? Thanks.

File: Sources/OAuthSwiftHTTPRequest.swift [hunk 1]
```diff
@@ -431,7 +431,7 @@ public struct URLSessionFactory {
     public static let `default` = URLSessionFactory()
 
     public var configuration = URLSessionConfiguration.default
-    public var queue = OperationQueue.main
+    public var queue: OperationQueue? = OperationQueue.main
```
Comment (phimage) [inline Sources/OAuthSwiftHTTPRequest.swift]: This change is not related to the subject

File: OAuthSwift.podspec [hunk 2]
```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
     'Eric Marchand' => 'eric.marchand.n7@gmail.com'
   }
   s.source = { git: 'https://github.com/OAuthSwift/OAuthSwift.git', tag: s.version }
-  s.source_files = 'Sources/*.swift'
+  s.source_files = 'Sources/*.swift', 'Sources/Handler/*.swift'
```
Comment (phimage) [inline OAuthSwift.podspec]: I think there is a recursive notation `**/*.swift`
